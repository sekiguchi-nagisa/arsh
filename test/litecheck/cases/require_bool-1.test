# RUN: echo hello
# CHECK: hello
#REQUIRE:   34 + 34