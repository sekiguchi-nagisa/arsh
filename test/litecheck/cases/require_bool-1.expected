require_bool-1.test:3: [error] REQUIRE directive must be boolean expression: `  34 + 34'